pkg/tests/ossm/setup.go

```diff
@@ -59,13 +59,30 @@ func SetupNamespacesAndControlPlane() {
 	if env.Getenv("NIGHTLY", "false") == "true" {
 		installNightlyOperators()
 	}
-
+	versionTemplates := GetSMCPTemplates()
+	smcpVersion := env.GetDefaultSMCPVersion()
+	template, ok := versionTemplates[smcpVersion]
+	if !ok {
+		log.Log.Errorf("Unsupported SMCP version: %s", smcpVersion)
+		return
+	}
 	util.ShellMuteOutputError(`oc new-project %s`, meshNamespace)
-	util.KubeApplyContents(meshNamespace, util.RunTemplate(smcpV23_template, smcp))
+	util.KubeApplyContents(meshNamespace, util.RunTemplate(template, smcp))
 	util.KubeApplyContents(meshNamespace, smmr)
 	util.Shell(`oc -n %s wait --for condition=Ready smcp/%s --timeout 180s`, meshNamespace, smcp.Name)
 	util.Shell(`oc -n %s wait --for condition=Ready smmr/default --timeout 180s`, meshNamespace)
 	if ipv6 == "true" {
 		log.Log.Info("Running the test with IPv6 configuration")
 	}
 }
+
+// Initialize a default SMCP and SMMR
+func SetupOnlyNamespaces() {
+	log.Log.Info("Setting up only namespaces")
+	createNamespaces()
+	if env.Getenv("NIGHTLY", "false") == "true" {
+		installNightlyOperators()
+	}
+	util.ShellMuteOutputError(`oc new-project %s`, meshNamespace)
```

Comment:
this is the same code that's in `SetupNamespacesAndControlPlane`, so it would make sense to extract it into a separate function

Reply:
Done